mappings/net/minecraft/util/math/random/AsyncRandom.mapping

```diff
@@ -1,4 +1,5 @@
-CLASS net/minecraft/class_7384 net/minecraft/util/math/random/BlockingSimpleRandom
+CLASS net/minecraft/class_7384 net/minecraft/util/math/random/AsyncRandom
+	COMMENT A random that can be shared by multiple threads safely.
```

Comment:
Is this really async, or just "async-safe"?

Reply:
This probably needs to be `ThreadSafeRandom` then